mantidproject/mantid
# Fix sphinx warnings - Andrei's branch

**Description of work.**

See  also and a few doc improvements.  This is @AndreiSavici 's work, I just created the PR

**To test:**

Code review of changes and check the affected pages after building the docs

No associated issue

Does this update require release notes?


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
